src/app/gigs/gigs-comms.tsx

```diff
@@ -109,14 +109,14 @@ export const GigsComms = ({
   });
 
   useEffect(() => {
-    toast({
-      title: 'Unable to fetch contract information',
-      description: 'If the problem persists, please contract support',
-    });
+    if (isGetContractError === true) {
```

Comment:
```suggestion
    if (isGetContractError) {
```

Right? since this is a boolean?